configuration/uppmax-genomes.config

```diff
@@ -23,13 +23,8 @@ params {
       snpeffDb      = "GRCh37.75"
       vardictHome   = "/sw/apps/bioinfo/VarDictJava/1.4.5/milou/VarDictJava/"
 
-      genomeAmb     = "${genome}.amb"
-      genomeAnn     = "${genome}.ann"
-      genomeBwt     = "${genome}.bwt"
+      bwaIndex      = "${genome}.*"  // this includes a few extra files not needed by BWA
```

Comment:
Does that work? Does `file()` support this type of wildcard? Also, we need the `.alt` file.